tests/network/vmi_networking.go

```diff
@@ -805,6 +780,31 @@ var _ = Describe(SIG("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:
 	})
 }))
 
+func newfedoraMasqueradeIPv6VMI(ports []v1.Port, ipv6NetworkCIDR string) (*v1.VirtualMachineInstance, error) {
```

Comment:
If you want to follow the convention:

```suggestion
func newFedoraMasqueradeIPv6VMI(ports []v1.Port, ipv6NetworkCIDR string) (*v1.VirtualMachineInstance, error) {
```

Reply:
Thanks. 
Commit message and PR desc too please.